packages/expo-modules-core/ios/Services/EXReactFontManager.m

```diff
@@ -94,6 +110,29 @@ + (void)initialize
                                  class_getClassMethod(rtcClass, exUpdate));
 }
 
+// A utility function used for swizzling UIFont for new architecture
++ (void)registerFontsForUIFont
+{
+  Class uiFont = [UIFont class];
+  SEL uiUpdate = @selector(fontWithName:size:);
+  SEL exUpdate = @selector(EXfontWithName:size:);
+
+  Method originalMethod = class_getClassMethod(uiFont, uiUpdate);
+  Method swizzledMethod = class_getClassMethod(uiFont, exUpdate);
+
+  method_exchangeImplementations(originalMethod, swizzledMethod);
+}
```

Comment:
Maybe let's have just one method but with `#ifdef RN_FABRIC_ENABLED` to separate these implementations